testplan/web_ui/testing/src/Report/BatchReport.js

```diff
@@ -64,7 +64,17 @@ class BatchReport extends React.Component {
         }),
         1500);
     } else {
-      axios.get(`/api/v1/reports/${uid}`)
+      const axiosConfig = {};
+      // allow overriding where we fetch the report from in deveopment, see src/Common/Home.jsx:12
+      if(process.env.NODE_ENV === "development" && typeof process.env.REACT_APP_COUCHDB_HOST !== "undefined") {
```

Comment:
It's getting a little confusing how we have the branch above checking for the special _dev report UID, and now a different branch checking for development env variables. I'm not sure we need the special _dev branch any more really. But if you want to keep it, it might be worth splitting into  a couple of separate functions to cover the special _dev page and the regular report.

Reply:
I've actually set it up now to just understand `/whatever/route?dev=true`. React Router ignores all query params by default so this way is less disruptive.

In my next push I'll merge the `if (uid === "_dev")` into `?dev=true`.